content/en/docs/cloud/edgectl/_index.md

```diff
@@ -147,7 +147,7 @@ Available Commands:
   microservice        Get details of an app microservice
   microservices       List app microservices
   node                Get node details
-  node-install-script Get the node installation script for a project
+  node-register-script Get the node registration script for a project
```

Comment:
node-register-script command indentation need to be corrected
<img width="863" alt="Screenshot 2023-03-08 at 12 03 40 AM" src="https://user-images.githubusercontent.com/35979902/223517408-4118095d-95fc-4deb-9d3c-b2fb86cbd9ea.png">

Reply:
Per @neilotoole we are moving to node-register-script replacing installation-script. I will let him confirm before committing this change.